contributing.md

```diff
@@ -1,6 +1,13 @@
 ## Contributing to HeAT
 
-*These contribution guidelines are inspired by NumPy's contribution guidelines of the SciPy community.*
+Thanks for your interest in contributing to HeAT!
```

Comment:
"Thank you"....
And maybe add something: "We appreciate your effort" ?

Reply:
Done